Gems/Prefab/PrefabDependencyViewer/Code/Source/PrefabDependencyViewerEditorSystemComponent.cpp

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzToolsFramework/API/ViewPaneOptions.h>
+#include <PrefabDependencyViewerEditorSystemComponent.h>
+#include <MainWindow.h>
+#include <QMenu>
+#include <LyViewPaneNames.h>
+#include <PrefabDependencyTree.h>
+
+namespace PrefabDependencyViewer
+{
+    using Outcome = AZ::Outcome<PrefabDependencyTree, const char*>;
+
+    InstanceEntityMapperInterface* PrefabDependencyViewerEditorSystemComponent::s_prefabEntityMapperInterface = nullptr;
+    PrefabSystemComponentInterface* PrefabDependencyViewerEditorSystemComponent::s_prefabSystemComponentInterface = nullptr;
+    PrefabPublicInterface* PrefabDependencyViewerEditorSystemComponent::s_prefabPublicInterface = nullptr;
+
+    const char* PrefabDependencyViewerEditorSystemComponent::s_prefabViewerTitle = "Prefab Dependencies Viewer";
```

Comment:
This can be a `static constexpr` probably.

Reply:
Resolved